apkid/rules/apk/packers.yara

```diff
@@ -114,6 +115,23 @@ rule dexprotector : packer
     is_apk and 1 of ($encrptlib_*) and $encrptcustom
 }
 
+rule tencentbugly : packer
+{
+ /**
+ * Tencent Bugly v1.x/v2.x (MultiDex Packer)
+ **/
+ 
+  meta:
+    author = "Jasi2169"
+    description = "Tencent Bugly"
+    
+  strings:
+    $main_lib = "libBugly.so"
+    
+  condition:
+    apk and $main_lib
```

Comment:
This seems a little week. Any way to fingerprint the actual unpacking code? Also, I think `apk` is causing the rules to fail to compile, hence the failing build. I think it's supposed to be `is_apk`. You can test the rules yourself by running `./prep_release.py`.